### PR TITLE
Fix display pixelclk active

### PR DESCRIPTION
**Pull request by ngphibang**
On the RT1170-EVKB, the framebuffer displays incorrectly (shifted by about 1/4 of the screen width). This is because pixelclk-active was set to 1 which corresponds to kELCDIF_DriveDataOnRisingClkEdge.
According to the HX8394 datasheet, it should be set to 0 which corresponds to kELCDIF_DriveDataOnFallingClkEdge.

### CODE REVIEW

**Comment by trunghieulenxp** [comment]
Hi @danieldegrasse, 

Could you take a look, please ? 

Thanks.

File: boards/shields/rk055hdmipi4m/rk055hdmipi4m.overlay
Comment (danieldegrasse) [inline boards/shields/rk055hdmipi4m/rk055hdmipi4m.overlay:1]: The RK055HDMIPI4M has a RM68200 controller- this change is still correct for the shield though: 
image

We just need to update the commit message

Comment (trunghieulenxp) [reply boards/shields/rk055hdmipi4m/rk055hdmipi4m.overlay:1]: Hi @danieldegrasse,

i have updated the commit message.

Thanks

**Review by danieldegrasse — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Comment by dleach02** [comment]
@erwango, can you review and approve so we can get this merged?

**Review by erwango — APPROVED** [review]

**Pull request merged**